lib/login.ts

```diff
@@ -112,7 +112,7 @@ export interface AuthResponse {
    */
   credentials: TokenCredentialsBase;
   /**
-   * @property {Array<LinkedSubscription>} [subscriptions] List of associated subscriptions.
+   * @property {Array<LinkedSubscription>} [subscriptions] List of associated subscriptions. Caution: This list may be empty if the credential was created without a domain i.e tenantId when using personal accounts.
```

Comment:
I see now that this can be confusing as one might try to see where to find the `tenantId` property.

```suggestion
   * @property {Array<LinkedSubscription>} [subscriptions] List of associated subscriptions. Caution: This list may be empty if the credential was created without setting the domain to the tenant id when using personal accounts.
```